infra/src/main/java/com/redhat/service/bridge/infra/auth/AbstractOidcClient.java

```diff
@@ -17,17 +17,17 @@
 
 public abstract class AbstractOidcClient {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractOidcClient.class);
+    protected static final Logger LOGGER = LoggerFactory.getLogger(AbstractOidcClient.class);
```

Comment:
Is there any reason to make LOGGER protected? Child classes should probably have their own loggers anyway.